src/utils.js

```diff
@@ -0,0 +1,6 @@
+/* @flow */
+
+// Node-style callback
+// FIXME: Could not find a way to define a Callback<T> type with multiple
+//        signatures.
+export type Callback = (?Error, any) => void;
```

Comment:
But when I use it, flow complains about `result` being nullable, even in the `else` branch of an `if (err)` check.